### PR TITLE
Remove deprecated funtions etc.

### PR DESCRIPTION
**Pull request by milliams**
This is hopefully a completely safe PR as it only removes things that have been deprecated or not working for many years (since at least 2010). They have all had deprecation warnings printed and many aren't even user-reachable.

### CODE REVIEW

**Comment by alexanderrichards** [comment]
Looks safe to me

File: python/Ganga/GPIDev/Lib/Job/Job.py [hunk 1]
```diff
@@ -1911,10 +1911,6 @@ def doit(f):
             logger.debug("Remove Err: %s" % str(err))
             pass
 
-    def fail(self, force=False):
```
Comment (rob-c) [inline python/Ganga/GPIDev/Lib/Job/Job.py]: This needs to be removed from _export_methods too for at least the Job class

Comment (milliams) [inline python/Ganga/GPIDev/Lib/Job/Job.py]: Thanks @rob-c I'd missed this. I guess this is another reason to put together a patch to move to using the `@export` decorator I developed.

**Comment by milliams** [comment]
If there are no further comment I'd like to merge this.

**Comment by alexanderrichards** [comment]
none from me

**Pull request merged**